xAPI.md

```diff
@@ -3280,10 +3299,10 @@ requested by voidedStatementId.
 Statement when retrieving Statements using explicit or implicit time or sequence based retrieval,
 unless they themselves have been voided, as described in
 [the section on filter conditions for StatementRefs](#queryStatementRef). This includes the
-voiding Statement, which cannot be voided. Reporting tools can identify the presence and
+voiding Statement, which cannot be voided. Clients can identify the presence and
 statementId of any voided Statements by the target of the voiding Statement. 
 
-* Reporting tools wishing to retrieve voided Statements SHOULD request these individually by 
+* Clients wishing to retrieve voided Statements SHOULD request these individually by 
```

Comment:
I haven't checked Blame, but I imagine this was a 'should' that became a 'SHOULD' when it should have become a 'can'. I'll look at it within this PR.